aries-site/src/components/seo/Meta.js

```diff
@@ -11,6 +11,18 @@ export const Meta = ({ title, description, canonicalUrl, socialImageUrl }) => {
   const segment = 'corporate';
   const lifecycle = 'support';
   const pageContent = 'products';
+  const csp = `default-src 'self' 'unsafe-eval'; 
+  style-src 'self' *.hpe.com/hfws-static/5/css/ 'unsafe-inline';
+  connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://www.google-analytics.com https://www.github.com/grommet/ https://eyes.applitools.com *.hpe.com/hpe/api/;
+  media-src 'self' https://d3hq6blov2iije.cloudfront.net/media/HPE+Design+System-v3.mp4;
+  img-src 'self' https://www.google-analytics.com https://images.unsplash.com/ http://s.gravatar.com/avatar/ *.hpe.com/hfws-static/5/;
+  ${
+    // in dev mode, use script-src-elem, otherwise we run into issues with
+    // react hot reloader
+    process.env.NODE_ENV !== 'production' ? 'script-src-elem' : 'script-src'
```

Comment:
Doesn't this mean in dev mode the global header doesn't show up in safari and FF? Seems like we might instead remove the 'unsafe-eval' in dev mode?

Reply:
Yes I think that's the better approach. matt was still seeing some console errors. going to make an update.